utils.py

```diff
@@ -0,0 +1,194 @@
+#%%
+
+
+import geojson
+import xarray as xr
+import re
+import json
+import openeo
+from typing import Union
+from pathlib import Path
+import pytest
+
+def extract_test_geometries(filename) -> dict:
+    """
+    Read the geometries from a test file that is stored within the project
+    :param filename: Name of the GeoJSON file to read
+    :return: GeoJSON Geometry collection
+    """
+    path = f'./geofiles/{filename}'
+    print(f'Reading geometries from {path}')
+
+    with open(path) as f:
+        geometry_collection = geojson.load(f)
+        f.close()
```

Comment:
Don't call `f.close()`: that's the point of using `with`: it closes the file automatically